libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -144,4 +197,103 @@
  */
 - (void)log:(nonnull Class)class level:(DDLogLevel)level format:(nonnull NSString *)format, ...;
 
+/**
```

Comment:
They will be set for the proper component.  The key is the `sharedInstance` override.  Each of our established components (like `SFSDKAnalyticsLogger` is here) override this method, to return their own component.  Same for the log messages.

The only time that they'd go to the default is if you e.g. did `[SFSDKLogger setLogLevel:]`, which IMO is also intuitive—the default is simply the base class.  Thoughts?